app/controllers/result_tables_controller.rb

```diff
@@ -145,8 +147,12 @@ def convert_contents_to_utf8
     end
   end
 
+  def check_create_permissions
+    render_403 unless can_create_my_module_results?(@my_module)
```

Comment:
may be can_create_results? ?

Reply:
It is part of my_module permissions, we have this format for permissions there - _action__**my_module**__objects_